OxfordBodleian/Juel-Jensen/BDLaethd18.xml

```diff
@@ -185,12 +502,23 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
          </xi:include>
       </encodingDesc>
         <profileDesc>
-            <creation/>
-            <abstract>
-                <p/>
-            </abstract>
+            <particDesc>
+                <listPerson>
+                    <person>
+                        <persName ref="PRS12944AmhaSe" role="scribe"/>
```

Comment:
This list is not really necessary, when you use and mark up with the respective role all IDs in the record, it will extracted automatically.